integrations/db/ojdbc/pom.xml

```diff
@@ -44,10 +44,6 @@
             <artifactId>ojdbc11-production</artifactId>
             <type>pom</type>
             <exclusions>
```

Comment:
Note: I deliberately removed a non-functional `exclusion` that was excluding the `ucp` artifact, which does not appear in the `ojdbc11-production:pom` artifact's dependencies (thus having no effect). Since roughly February users have had `ucp11` on their classpath, since it was not being excluded. This PR preserves `ojdbc11-production:pom`'s set of dependencies, with the sole exception of `xmlparserv2` (we continue to exclude that and include `xmlparserv2_sans_jaxp_services` in its place as before).

Reply:
Per instructions I will be re-introducing an `<exclusion>`, but this time one that will exclude `ucp*` (most particularly `ucp11`). This means indirect consumers of this `pom`-typed artifact who might have been accustomed to having `ucp11` on their classpath will now have to include `ucp11` explicitly. We need to update the examples to make this easy to understand.